config/brakeman.ignore

```diff
@@ -1,5 +1,28 @@
 {
   "ignored_warnings": [
+    {
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "c6be575b25f7d0b6a3a83da81c1680f867bc8262b47ae4edd92b93b0bd770667",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "modules/travel_pay/app/services/travel_pay/documents_client.rb",
+      "line": 101,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "connection(:server_url => Settings.travel_pay.base_url).delete(\"api/v1/claims/#{claim_id}/documents/#{document_id}\")",
```

Comment:
Not sure how this got in here, we should probably not check this file in but....

Brakeman has a point - it may be worth sanitizing these IDs.